mautic/mautic
# Clickable email stats feature

| Q  | A
| --- | ---
| Bug fix? | N
| New feature? | Y
| Related user documentation PR URL | N/A
| Related developer documentation PR URL | N/A
| Issues addressed (#s or URLs) | N/A
| BC breaks? | N/A
| Deprecations? | N/A

#### Description:
This PR makes it possible to click on an email stat and see a list of contacts that the email was sent to, that are pending or queued to send or that read the email. 

#### Steps to test this PR:
1. Apply this PR
2. Go to Channels/Emails
3. Click on any stat to see the list of contacts for that specific stat
